dev/roles/base/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+
+- name: Add ZMQ key
+  apt_key:
+    url: http://download.opensuse.org/repositories/network:/messaging:/zeromq:/release-stable/xUbuntu_14.04/Release.key
+    state: present
+
+- name: Add ZMQ ppa
+  apt_repository:
+    repo: deb http://download.opensuse.org/repositories/network:/messaging:/zeromq:/release-stable/xUbuntu_14.04/ ./
```

Comment:
As the vagrant box is now debian jessie, you should change this to Debian_8.0 from http://download.opensuse.org/repositories/network:/messaging:/zeromq:/release-stable/

Reply:
@mvonmaltitz Can you update the ZeroMQ repo to point to Debian 8?

Reply:
Didn't get the notification about this comment. Changes applied.